### PR TITLE
Story#668

### PR DESCRIPTION
**Pull request by rejneesh1**

### CODE REVIEW

File: src/pages/programs/program-solution/solution-action/solution-action.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+<!-- <ion-header>
+  <ion-navbar>
+    <ion-title>solutionAction</ion-title>
+  </ion-navbar>
+</ion-header> -->
+
+<ion-content>
+  <ion-list>
+    <button ion-item (click)="presentConfirm()">Remove from home</button>
```
Comment (kiranharidas187) [inline src/pages/programs/program-solution/solution-action/solution-action.html]: en.json

File: src/pages/programs/program-solution/solution-action/solution-action.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <button ion-item (click)="presentConfirm(true)" *ngIf="isAPrivateProgram">Delete</button>
```
Comment (kiranharidas187) [inline src/pages/programs/program-solution/solution-action/solution-action.html]: en.json

File: src/pages/programs/program-solution/solution-action/solution-action.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            handler: () => {
+              this.delete();
+            },
+          },
+        ],
+      });
+    } else {
+      alert = this.alertCtrl.create({
+        // title: "Confirm purchase",
+        message: "The solution will no longer appear on the home screen",
```
Comment (kiranharidas187) [inline src/pages/programs/program-solution/solution-action/solution-action.ts]: all text put in en.json

File: src/pages/trash/trash-action/trash-action.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+      buttons: [
+        {
+          text: "Cancel",
+          handler: () => {
+            this.viewCtrl.dismiss();
+          },
+        },
+        {
+          text: "Delete Permanently",
```
Comment (kiranharidas187) [inline src/pages/trash/trash-action/trash-action.ts]: en.json

File: src/providers/trash/trash.ts [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import { HttpClient } from "@angular/common/http";
```
Comment (kiranharidas187) [inline src/providers/trash/trash.ts]: use of this import

File: src/pages/programs/program-solution/solution-action/solution-action.html (hunk 1 above)
Comment (rejneesh1) [reply src/pages/programs/program-solution/solution-action/solution-action.html]: done

File: src/pages/programs/program-solution/solution-action/solution-action.html (hunk 2 above)
Comment (rejneesh1) [reply src/pages/programs/program-solution/solution-action/solution-action.html]: done

File: src/pages/programs/program-solution/solution-action/solution-action.ts (hunk 3 above)
Comment (rejneesh1) [reply src/pages/programs/program-solution/solution-action/solution-action.ts]: done

File: src/pages/trash/trash-action/trash-action.ts (hunk 4 above)
Comment (rejneesh1) [reply src/pages/trash/trash-action/trash-action.ts]: done

File: src/providers/trash/trash.ts (hunk 5 above)
Comment (rejneesh1) [reply src/providers/trash/trash.ts]: removed

**Pull request merged**